### ISSUE TITLE
multiple run  mounted()

### ISSUE THREAD

**Issue by mehrdadsalimii** [Akryum/vue-cli-plugin-ssr#170]
hi guys.

All components run two times. I run console.log('test') in all components it is a print twice.

So far no one has encountered this problem
![Screenshot from 2019-12-29 12-10-21](https://user-images.githubusercontent.com/29326138/71554443-53c05100-2a34-11ea-8bb5-3a7d4721143f.png)
?

**Comment by darkiron** [comment]
Same probleme 
Some issue related this fact about webpack and vue-router

Visible on simple vue-cli and ssr-plugin. 

Any ideas ?

**Comment by Zacaria** [comment]
It seems like the renderScripts function outputs the bundle as well as being output by replacing the  placeholder

When you remove the {{{ renderScripts() }}} in the index.ssr.html your app is mounted only once.

But I'm actually looking to understand the possible side effect of removing the renderScripts

**Comment by tarasbilohan** [comment]
Duplicate of #158
